Description: Remove hardcoded prefix
 The Makefile your uses an hardcoded prefix so we can't reassign it and build in a chroot. 
Author: Pierre Rudloff <contact@rudloff.pro>
Forwarded: https://sourceforge.net/p/rarcrack/bugs/13/
Last-Update: 2013-08-14

--- a/Makefile
+++ b/Makefile
@@ -1,16 +1,21 @@
 CC=gcc
-PREFIX=/usr
-DOCDIR=${PREFIX}/share/doc/rarcrack
+PREFIX?=/usr
+DOCDIR=${DESTDIR}/${PREFIX}/share/doc/rarcrack
+BINDIR=${DESTDIR}/${PREFIX}/bin
+CPPFLAGS:=$(shell dpkg-buildflags --get CPPFLAGS)
+CFLAGS:=$(shell dpkg-buildflags --get CFLAGS)
+CXXFLAGS:=$(shell dpkg-buildflags --get CXXFLAGS)
+LDFLAGS:=$(shell dpkg-buildflags --get LDFLAGS)
 
 all:
-	${CC} -pthread rarcrack.c `xml2-config --libs --cflags` -O2 -o rarcrack
+	${CC} -pthread rarcrack.c `xml2-config --libs --cflags` ${CPPFLAGS} ${LDFLAGS} -O2 -o rarcrack
 clean:
 	-rm *.o rarcrack
 install:
-	install -s rarcrack ${PREFIX}/bin
+	-mkdir -p ${BINDIR}
+	install -s rarcrack ${BINDIR}
 	-mkdir -p ${DOCDIR}
 	chmod 755 ${DOCDIR}
-	install -m 644 -t ${DOCDIR} CHANGELOG LICENSE README README.html RELEASE_NOTES
+	install -m 644 -t ${DOCDIR} CHANGELOG README README.html RELEASE_NOTES
 uninstall:
-	-rm ${PREFIX}/bin/rarcrack
-
+	-rm ${BINDIR}/rarcrack
